server/pubsub/redis_query_results.go

```diff
@@ -188,6 +195,7 @@ func (r *redisQueryResults) ReadChannel(ctx context.Context, query fleet.Distrib
 		wg.Wait()
 		psc.Unsubscribe(pubSubName) //nolint:errcheck
 		conn.Close()
+		level.Info(logger).Log("msg", "proper close of Redis connection in ReadChannel", "connection", fmt.Sprintf("%p", conn))
```

Comment:
Let me know if you'd rather have those as Debug logs, but since it should happen only once per live query (or is it once per live query consumer?) in any case I think it should be rare enough not to become noise?